src/platform_impl/linux/x11/window.rs

```diff
@@ -431,7 +433,7 @@ impl UnownedWindow {
 
             // Select XInput2 events
             let mask = {
```

Comment:
Is the extra scope necessary/useful here?

Reply:
Removed the scope